### PR TITLE
Add templating for proper localization

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

**Comment by RA489** [comment]
@minikube-bot OK to test

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ assumes you have already installed one of the VM drivers: virtualbox/parallels/v
 
 // runStart handles the executes the flow of "minikube start"
 func runStart(cmd *cobra.Command, args []string) {
-	console.OutStyle(console.Happy, "minikube %s on %s (%s)", version.GetVersion(), runtime.GOOS, runtime.GOARCH)
+	console.OutTemplateStyle(console.Happy, "minikube {{.version}} on {{.os}} ({{.arch}})", console.Arg{"version": version.GetVersion(), "os": runtime.GOOS, "arch": runtime.GOARCH})
```
Comment (tstromberg) [inline cmd/minikube/cmd/start.go]: As we will be calling this all over the code base, can the name get any shorter? How about `OutT` and `ErrT`?

File: pkg/minikube/console/style.go [hunk 2]
```diff
@@ -130,29 +131,46 @@ func lowPrefix(s style) string {
...
 	}
 
 	if !useColor {
-		return applyPrefix(lowPrefix(s), out)
+		return applyPrefix(lowPrefix(s), format)
+	}
+	return applyPrefix(s.Prefix, format)
+}
+
+func applyTemplateFormatting(style StyleEnum, useColor bool, format string, b map[string]interface{}) string {
```
Comment (tstromberg) [inline pkg/minikube/console/style.go]: just curious - what does `b` stand for?

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (sharifelgamal) [reply cmd/minikube/cmd/start.go]: Absolutely. Done.

File: pkg/minikube/console/style.go (hunk 2 above)
Comment (sharifelgamal) [reply pkg/minikube/console/style.go]: It was `b` when I was testing OutStyle being one function for both versions. Changed back to `a`.

**Pull request merged**